src/components/UserInfo.tsx

```diff
@@ -0,0 +1,23 @@
+import { User } from '../types/User';
+
+type Props = {
+  userData: User | null
+};
+
+export const UserInfo: React.FC<Props> = ({ userData }) => (
+  <>
+    {userData !== null
+      ? (
+        <p>
+          {userData.name}
+          {' - '}
+          {userData.email}
+        </p>
+      )
+      : (
+        <p>
+          No user data
+        </p>
+      )}
```

Comment:
```suggestion
      <p>
        {userData.name}
        {' - '}
        {userData.email}
      </p>
    )}
```
Можна рядок 'No user data' рендерити рівнем вище (якщо немає item.user). ТОді не доведеться прописувати null для типу